spec/data-types-and-values.html

```diff
@@ -22,7 +22,35 @@ <h1>
           <dd>It converts _argument_ to a String.</dd>
         </dl>
         <emu-alg>
-            1. Return *"[object Record]"*.
+            1. Let _list_ be _argument_.[[Fields]].
+            1. Let _strings_ be a new empty List.
+            1. For each Record { [[Key]], [[Value]] } _kv_ of _list_, do
+              1. Let _key_ be _kv_.[[Key]].
+              1. Let _kvString_ be QuoteJSONString(_key_).
+              1. Set _kvString_ to be the string-concatenation of _kvString_ and *": "*.
+              1. Let _value_ be _kv_.[[Value]].
+              1. Set _kvString_ to be the string-concatenation of _kvString_ and StringifyRecordValue(_value_).
+              1. Append _kvString_ to the end of _strings_.
+            1. Let _ret_ be *"#{ "*.
+            1. Set _ret_ to be the string-concatenation of _ret_ and ListJoin(_strings_).
+            1. Set _ret_ to be the string-concatenation of _ret_ and *" }"*.
+            1. Return _ret_.
+        </emu-alg>
+      </emu-clause>
+
+      <emu-clause id="sec-immutable-types-stringify-record-value" type="abstract operation">
+        <h1>
+          <ins>
+            StringifyRecordValue (
+              _value_: an ECMAScript language value, but not an Object,
```

Comment:
```suggestion
              _value_: a primitive value,
```
(this may require adding a `<dfn>` to the definition in https://tc39.es/ecma262/#sec-ecmascript-overview at spec PR time)